src/MicroOrm.Dapper.Repositories/SqlGenerator/SqlGenerator.GetSelect.cs

```diff
@@ -341,7 +341,7 @@ string ProjectionFunction(SqlPropertyMetadata p)
             {
                 return !string.IsNullOrEmpty(p.Alias)
                     ? $"{tableName}.{(useQuotation ? p.ColumnName : p.CleanColumnName)} AS {p.PropertyName}"
-                    : $"{tableName}.{(useQuotation ? p.ColumnName : p.CleanColumnName)}";
+                    : $"{tableName}.[{(useQuotation ? p.ColumnName : p.CleanColumnName)}]";
```

Comment:
I don't think that is work with MySql, PostgreSql and etc

Reply:
`UseQuotationMarks` already add `[]` for MS SQL. So you need fix your issue with [UseQuotationMarks](https://github.com/phnx47/dapper-repositories/blob/a74062b8a6dd7153518f06fe8ac3e9b5218cabb2/src/MicroOrm.Dapper.Repositories/Config/MicroOrmConfig.cs#L18)